cmd/agent/app/agent.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2021 The Karmada Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I'm sorry i forgot to submit the comments made 3 days ago.

Reply:
Thanks, google/addlicense only support comment by '//' for .go file(https://github.com/google/addlicense/blob/4caba19b7ed7818bb86bc4cd20411a246aa4a524/main.go#L296), I will look for another tool or way to fix this.

Reply:
> Thanks, google/addlicense only support comment by '//' for .go file

I'm surprised, I thought it could be configurable.

>  I will look for another tool or way to fix this.

Thanks.  I'm not saying that the `/*` and `*/` is not acceptable, but we can see if there are other options.

Reply:
Fixed!